main.py

```diff
@@ -1,7 +1,6 @@
 import os, json, werkzeug
 from flask import Flask, request, current_app
-from credit import Credits
-
+from credit import Credits, logger
```

Comment:
Don't import `logger`, create one here in the same way as you do there.

Reply:
Done.